sequel-rails.gemspec

```diff
@@ -20,20 +21,20 @@ Gem::Specification.new do |s|
   s.rdoc_options = ['--charset=UTF-8']
   s.license = 'MIT'
 
-  s.required_ruby_version = ">= 2.2.2"
-  s.required_rubygems_version = ">= 1.8.11"
+  s.required_ruby_version = '>= 2.2.2'
+  s.required_rubygems_version = '>= 1.8.11'
 
   s.add_dependency 'activemodel', '>= 4.0.0'
   s.add_dependency 'railties', '>= 4.0.0'
   s.add_dependency 'actionpack', '>= 4.0.0'
-  s.add_dependency 'sequel', ['>= 3.28', '< 5.0']
+  s.add_dependency 'sequel', ['>= 3.28', '<= 5.0']
```

Comment:
I don't think we want to lock this in to `<= 5.0`. Sequel is already on 5.1.0`. Possibly `< 6.0` instead?

Reply:
@Jesterovskiy I opened a PR to fix this specific issue https://github.com/Jesterovskiy/sequel-rails/pull/1